### ISSUE TITLE
Uncaught TypeError: Cannot read property 'shift' of null(…)

### ISSUE THREAD

**Issue by andy00087** [AGMStudio/jquery.countup.js#1]
Hi AGMStudio,
I love this project but I did a test and I got this error:

**Uncaught TypeError: Cannot read property 'shift' of null(…)**

This error appear in the console every time I scroll up and down the page.
Another problem (perhaps related to this) and that the counter starts from scratch every time I see the counter on the screen during scrolling.

Best Regards,
Andrea

**Comment by mhsohag11** [comment]
Yes andy00087  , I also faced same problem in my project . Any solution have you found ??? 
Please , share this if you solve this problem !!!

**Comment by saddamwp** [comment]
Hello 
mhsohag11

can you follow this link: https://github.com/bfintal/Counter-Up/issues/10
I hope you will be got problem solution.

**Comment by Alokmishra3751** [comment]
Solution for counterup shift null :
link :https://github.com/bfintal/Counter-Up/pull/46/commits/7a826a4912693fd3453316a2f1c92c4ad8e6e6cc

**Comment by KZeni** [comment]
Please see https://github.com/AGMStudio/jquery.countup.js/pull/2 as the formally-available fix for this issue.

Can't say whether or not this PR will be officially adopted (it should be if the author has enough interest to just give it a look and quickly approve this documented fix for a clear issue, but they might've fully abandoned this script at this point... again, can't say for sure at this point.) Hoping it is so people don't need to specifically reference this PR (https://github.com/AGMStudio/jquery.countup.js/pull/2) or the fork created to accommodate my PR (https://github.com/KZeni/jquery.countup.js/tree/patch-1) to have this script work properly on a reasonably-modern site for those that might need it (might've been used by someone else & don't want to fully swap out the script while wanting to fix this issue, this still does the job after fixing the one bug here, etc.)
